### PR TITLE
Blazorise 1.0 migration

### PR DESCRIPTION
**Pull request by stsrki**
This is still using the preview version but all migration steps are covered. Final v1.0 is coming next week.

PS. I got problems with making bundle css/js. Maybe someone else can make them?

### CODE REVIEW

**Comment by hikalkan** [comment]
I think @enisn can help on that.

**Comment by realLiangshiwei** [comment]
May related to: https://github.com/abpframework/abp/issues/8528

For Blazor WASM: Run `abp install-libs` command in the `.HostWithIds` project

**Comment by enisn** [comment]
Updating Volo.Abp.Cli to version `5.2.0-preview20220214` solves bundling issues for `dev` branch.

---

```bash
dotnet tool update -g Volo.Abp.Cli --add-source https://www.myget.org/F/abp-nightly/api/v3/index.json --version 5.2.0-preview20220221
```

**Comment by stsrki** [comment]
@enisn I'm running `abp bundle` under the project `abp\templates\app\aspnet-core\src\MyCompanyName.MyProjectName.Blazor`. When I start the `MyCompanyName.MyProjectName.Blazor` project I get the following error:

image

**Comment by enisn** [comment]
> @enisn I'm running `abp bundle` under the project `abp\templates\app\aspnet-core\src\MyCompanyName.MyProjectName.Blazor`. When I start the `MyCompanyName.MyProjectName.Blazor` project I get the following error:
> 
> image

You need to run `abp install-libs` command under **HttpApi.Host** folder.
in that case it's **HttpApi.HostWithIds**

**Comment by stsrki** [comment]
Thank you @enisn. That worked :)

**Comment by stsrki** [comment]
@enisn Do I need to push `yarn.lock` files?

**Comment by enisn** [comment]
> @enisn Do I need to push `yarn.lock` files?

Yes please, we usually push **yarn.lock** files together with **package.json**

File: templates/app/aspnet-core/src/MyCompanyName.MyProjectName.HttpApi.Host/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (NecatiMeral) [inline templates/app/aspnet-core/src/MyCompanyName.MyProjectName.HttpApi.Host/yarn.lock]: This project doesn't have any client packages, so this lockfile can be deleted.

Comment (stsrki) [reply templates/app/aspnet-core/src/MyCompanyName.MyProjectName.HttpApi.Host/yarn.lock]: done

**Review by enisn — APPROVED** [review]

**Review by realLiangshiwei — APPROVED** [review]

**Comment by realLiangshiwei** [comment]
@stsrki Is it ready to merge?

**Comment by stsrki** [comment]
Technically yes. But I would wait until the final 1.0 is released on NuGet.

**Review by enisn — APPROVED** [review]

**Comment by enisn** [comment]
Thanks @stsrki 🚀

---

I'll merge after CI passes

**Comment by stsrki** [comment]
> Thanks @stsrki 🚀

You're welcome 😊

**Comment by stsrki** [comment]
PS. keep in mind that the same steps need to be done on the Volo repository.

**Comment by enisn** [comment]
> PS. keep in mind that the same steps need to be done on the Volo repository.

Ok, I'm creating yet another issue for that thanks for reminding

**Comment by enisn** [comment]
# Migrating to Blazorise 1.0
Developers should follow **Migration** section of this documentation: https://blazorise.com/news/release-notes/100

*Edit: fixed URL

**Pull request merged**